peregrine/tracking.py

```diff
@@ -207,8 +207,7 @@ def __init__(self, params):
     self.code_phase = 0.0
     self.carr_phase = 0.0
     self.samples_per_chip = int(round(self.sampling_freq / self.chipping_rate))
-    self.sample_index = params['samples']['sample_index']
-    self.sample_index += self.acq.sample_index
+    self.sample_index = self.acq.sample_index
```

Comment:
Would it be possible to add 'sample_index' entry into 'params' dictionary and initialize it differently for L1C/A and L2C tracking channels:
L1C/A case: params['sample_index'] = params['samples']['sample_index']
L2C case: params['sample_index'] = 0 ?
Then this code would look like this:
self.sample_index = params['sample_index']